### PR TITLE
Added milestones to GitHub workflow details article for .NET docs team

### PR DESCRIPTION
**Pull request by IEvangelist**
Added milestones to GitHub workflow details article for .NET docs team

#### Preview

✔️ [Labels, projects, and milestones roadmap](https://review.docs.microsoft.com/en-us/contribute/dotnet/labels-projects?branch=pr-en-us-421)

### CODE REVIEW

**Comment by IEvangelist** [comment]
#assign: @Jim-Parker

**Comment by Jim-Parker** [comment]
Reassigning to @BillWagner since he's the primary author. Bill, can you review?

File: Contribute/dotnet/labels-projects.md [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+# Labels, projects, and milestones roadmap
 
-We also use [GitHub projects](https://github.com/dotnet/docs/projects) to organize sprints and other goal-oriented epics.
+The .NET docs team makes extensive use of [GitHub labels](https://github.com/dotnet/docs/labels) to organize our work. By filtering on combinations of labels, we can quickly focus on sections of interest on the [.NET docs website](https://docs.microsoft.com/dotnet). For example, we could filter to all of the open priority one `P1` issues with a query to [`is:open label:p1 repo:dotnet/docs`](https://github.com/pulls?q=is%3Aopen+label%3Ap1+repo%3Adotnet%2Fdocs).
+
+We use [GitHub projects](https://github.com/dotnet/docs/projects) to organize sprints and other goal-oriented epics. We also use [GitHub milestones](https://github.com/dotnet/docs/milestones) to track work. It is best to think of projects for planning (issues), and milestones for work (pull requests).
```
Comment (tdykstra) [inline Contribute/dotnet/labels-projects.md:14]: If we're going to mention milestones we should explain exactly how we use them.  I'm inclined to think of them as not important enough to mention here.

File: Contribute/dotnet/labels-projects.md [hunk 2]
```diff
@@ -108,18 +109,27 @@ Issues tagged for a specific release are noted with the `:checkered_flag: Releas
...
+- P3 - Low priority
 
-### What about the other labels?
+### What about the other labels
 
 There are many other labels used by the content teams to manage different classifications of issues. If you're not on the content team, you can ignore these other labels.
 
 ## Projects
 
+Projects are intended for planning purposes, where prioritized work is automated through a Kanban board. Projects should only ever contain GitHub issues, _not_ pull requests. Projects differ from milestones, in that milestones only contain pull requests.
```
Comment (tdykstra) [inline Contribute/dotnet/labels-projects.md:123]: Here too, I'd omit mention of milestones.

File: Contribute/dotnet/labels-projects.md [hunk 3]
```diff
@@ -108,18 +109,27 @@ Issues tagged for a specific release are noted with the `:checkered_flag: Releas
...
 We use projects in two ways:
 
-- Month YYYY project types: These are scrum boards for each month's working plan.
+- `Month YYYY` project types: These are Kanban boards for each month's working plan.
+  - Examples, [July 2020](https://github.com/dotnet/docs/projects/103), [August 2020](https://github.com/dotnet/docs/projects/117), and so on.
 - Long-running epics: These are used to organize tasks toward a goal when the work will occur over several months.
+  - Examples: [.NET 5 Wave - Reorganization](https://github.com/dotnet/docs/projects/105), [.NET Languages (.NET 5 wave)
+](https://github.com/dotnet/docs/projects/106), and so on.
+
+## Milestones
```
Comment (tdykstra) [inline Contribute/dotnet/labels-projects.md:133]: As noted earlier, I'd omit this or explain how they're used.

**Comment by Jim-Parker** [comment]
@IEvangelist are you still working on this PR?

File: Contribute/dotnet/labels-projects.md (hunk 1 above)
Comment (IEvangelist) [reply Contribute/dotnet/labels-projects.md:14]: The entire idea behind this update to the article was to add how milestones relate to our work. It was something that was unclear to me, and we need to be explicit about them.

File: Contribute/dotnet/labels-projects.md (hunk 2 above)
Comment (IEvangelist) [reply Contribute/dotnet/labels-projects.md:123]: I think it is important to differentiate between projects and milestones.

File: Contribute/dotnet/labels-projects.md (hunk 3 above)
Comment (IEvangelist) [reply Contribute/dotnet/labels-projects.md:133]: I do actually state how we use them.

File: Contribute/dotnet/labels-projects.md (hunk 1 above)
Comment (tdykstra) [reply Contribute/dotnet/labels-projects.md:14]: I'm fine with including mention of milestones, but if we mention them we should explain how and why we use them.

**Comment by IEvangelist** [comment]
> @IEvangelist David Pine FTE are you still working on this PR?

Hi @Jim-Parker, sorry. Yes, I have updated it.

File: Contribute/dotnet/labels-projects.md (hunk 3 above)
Comment (tdykstra) [reply Contribute/dotnet/labels-projects.md:133]: Where?

**Comment by Jim-Parker** [comment]
@IEvangelist just checking status. No rush if you're still working on it. let me know when it's ready to merge.

**Pull request merged**